### ISSUE TITLE
veriT interface sometimes hangs on :- _|_

### ISSUE THREAD

**Issue by gebner** [gapt/gapt#338]
On Stefan's machine and compile.logic.tuwien.ac.at, the verit unit test sometimes hangs on the "not prove bottom" example.

When running this test, we pass the following SMT-LIB code to veriT:
```lisp
(set-logic QF_UF)
(declare-sort S 0)
(assert (not false))
(check-sat)
```

The unit test reliably works for me on my machines; and I haven't yet been able to reproduce it just using veriT on the compile server.

Has anyone else seen this issue?  What java version are you using?  What veriT version?

**Comment by shetzl** [comment]
veriT --version says: "This is veriT, version dev". I have installed this from a tarball named "veriT-201410.tar.gz" but it may not be the official 201410-version. If I remember correctly, Pascal once sent us a version for proof generation.

java -version says:
java version "1.7.0_55"
OpenJDK Runtime Environment (IcedTea 2.4.8) (suse-8.36.1-x86_64)
OpenJDK 64-Bit Server VM (build 24.55-b03, mixed mode)

I tried reproducing this behaviour from the CLI by doing
gapt> veriT.getExpansionSequent( FSequent( Seq(), Seq( Bottom() ) ) )
but this worked fine in all of ~40 attempts.
